tests/test_structs.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+import pytest
+
+import optuna
+from optuna.distributions import LogUniformDistribution
+from optuna.distributions import UniformDistribution
+from optuna.structs import FrozenTrial
+from optuna.structs import TrialState
+
+if optuna.types.TYPE_CHECKING:
+    from typing import Any  # NOQA
+    from typing import Dict  # NOQA
+    from typing import List  # NOQA
+    from typing import Optional  # NOQA
```

Comment:
Please remove this line because `Optional` is not used in this file.

Reply:
Removed: https://github.com/pfnet/optuna/pull/464/commits/b6cf0c5d479cd6cb5e6f28b83fe40e5188acdffb